src/transport/tests/TestSessionManager.cpp

```diff
@@ -137,8 +134,8 @@ TEST_F(TestSessionManager, CheckSimpleInitTest)
 
     EXPECT_EQ(CHIP_NO_ERROR, fabricTableHolder.Init());
     EXPECT_EQ(CHIP_NO_ERROR,
-              sessionManager.Init(&inContext->GetSystemLayer(), &inContext->GetTransportMgr(), &gMessageCounterManager,
-                                  &deviceStorage, &fabricTableHolder.GetFabricTable(), sessionKeystore));
+              sessionManager.Init(&this->GetSystemLayer(), &this->GetTransportMgr(), &gMessageCounterManager, &deviceStorage,
```

Comment:
You can just call functions, you don't have to use `&this->`

Reply:
Updated in all places.